src/current/_includes/releases/v23.1/v23.1.7.md

```diff
@@ -0,0 +1,56 @@
+
+- The **Regions/Nodes** column on the [**Database** and **Database Details** pages](../v23.1/ui-databases-page.html) will now render properly. This column displays for clusters with more than 1 node, but is hidden for tenants. [105824][#105824]
+- Fixed a bug where the [**Job Details** page](../v23.1/ui-jobs-page.html#job-details) would flicker between the job details and a loading animation while a job is still executing. [106153][#106153]
+- Fixed a bug where the [**Key Visualizer** page](../v23.1/ui-key-visualizer.html) was crashing with an invalid conversion to `int` error. [106357][#106357]
+- Fixed a bug that caused an infinite re-render on the [**Key Visualizer** page](../v23.1/ui-key-visualizer.html) when a custom time period was selected. [106430][#106430]
+- Fixed a bug on the **SQL Activity**, [**Statements** page](../v23.1/ui-statements-page.html) where the database filter would not return results even when statements existed for the selected database. This is now fixed so that selecting a database filter from the filters menu on the [**Statements** page](../v23.1/ui-statements-page.html) will function as expected. [106434][#106434]
+- Fixed a bug where the [**Metrics** page](../v23.1/ui-overview-dashboard.html) was not updating automatically on rolling window options. [106766][#106766]
+- Fixed a bug where [statement diagnostics](../v23.1/ui-statements-page.html#diagnostics) on the [**Statements** page](../v23.1/ui-statements-page.html) were not always showing. This is now fixed with statement diagnostics displaying for the correct time period. [106776][#106776]
+- Fixed a bug where the **Sort** dropdown on the [**Network Latency** page](../v23.1/ui-network-latency-page.html) would not persist the selected value because a page reload was triggered. [107213][#107213]
+- Fixed a bug where the index recommendation on the [**Insights** page](../v23.1/ui-insights-page.html) would use the fully qualified table name to create an index name, which would cause an error due to the invalid syntax. Indexes can now be created directly from the DB Console without encountering this error. [107218][#107218]
+- Fixed a bug where [changefeeds](../v23.1/change-data-capture-overview.html) would fail when upgrading to version [v23.1.5](#v23-1-5) because the job record did not have a `clusterID` field set. [106399][#106399]
+- Fixed a bug where `UPDATE`, `UPSERT`, `DELETE` statements running concurrently with [`ALTER TABLE..ADD COLUMN`](../v23.1/alter-table.html#add-column) of a virtual computed column on the same table would fail. [107403][#107403]
+- Fixed a bug where [SQL statistics compaction jobs](../v23.1/show-jobs.html#show-automatic-jobs) would fail. This is now fixed by reducing the contention on the `system.statement_statistics` table. [107573][#107573]
+
+<div class="release-note-contributors" markdown="1">
+
+<h3 id="v23-1-7-contributors">Contributors</h3>
+
+This release includes 0 merged PRs by 11 authors.
```

Comment:
is this number correct? How come it shows 0 merged PRs?

Reply:
The reason is because the release note generation just pulled commits, since they were cherry picked. I will double check if want to simply add the PRs listed that contain those commits.

Reply:
Okay sounds good with me! For v23.1.6 which was also a cherry pick I manually counted the PRs in the list